### PR TITLE
Enable Travis testing

### PR DESCRIPTION
**Pull request by Chris00**

### CODE REVIEW

**Comment by Chris00** [comment]
See https://travis-ci.org/Chris00/oasis/builds/72039111 for a run.

**Comment by Chris00** [comment]
The run fails because of the tests.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+language: c
```
Comment (gildor478) [inline .travis.yml]: Not familiar with travis but why "language: c" ?

**Comment by gildor478** [comment]
Thanks a lot for the PR, looks very promising.

**Pull request merged**